_data/chains/eip155-1868.json

```diff
@@ -14,11 +14,28 @@
   "infoURL": "https://soneium.org",
   "chainId": 1868,
   "networkId": 1868,
-  "status": "incubating",
-  "explorers": [],
+  "status": "active",
+  "explorers": [
+    {
+      "name": "blockscout",
+      "url": "https://soneium.blockscout.com",
+      "icon": "blockscout",
+      "standard": "EIP3091"
+    },
+    {
+      "name": "OKLink",
+      "url": "https://www.okx.com/web3/explorer/soneium",
+      "standard": "EIP3091"
+    },
+    {
+      "name": "SlamVision",
+      "url": "https://soneium.slam.vision",
+      "standard": "EIP3091"
```

Comment:
```suggestion
      "standard": "none"
```

or remove the explorer completely

![image](https://github.com/user-attachments/assets/4ed555e2-02a7-442b-9d2c-ca3aee9ad56b)